src/contrib/cluster/Akka.Cluster.Sharding/ShardingSetup.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -----------------------------------------------------------------------
+//  <copyright file="ShardingSetup.cs" company="Akka.NET Project">
+//      Copyright (C) 2009-2025 Lightbend Inc. <http://www.lightbend.com>
+//      Copyright (C) 2013-2025 .NET Foundation <https://github.com/akkadotnet/akka.net>
+//  </copyright>
+// -----------------------------------------------------------------------
+
+using Akka.Actor.Setup;
+using Akka.Annotations;
+
+#nullable enable
+namespace Akka.Cluster.Sharding;
+
+[InternalApi]
+public class ShardingSetup: Setup
```

Comment:
This setup class will be used to inject a message adapter into both the `ShardRegion` and `Shard` actor during initialization